debian/copyright

```diff
@@ -0,0 +1,507 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: rdma-core
+Upstream-Contact: Doug Ledford <dledford@redhat.com>,
+                  Leon Romanovsky <Leon@kernel.org>
+Source: https://github.com/linux-rdma/rdma-core
+
+Files: *
+License: GPL-2 or BSD
+
+Files: debian/*
+Copyright: 2015-2016, Jason Gunthorpe <jgunthorpe@obsidianresearch.com>
+Copyright: 2016, Talat Batheesh <talatb@mellanox.com>
```

Comment:
I would use indentation instead of duplicate ```Copyright:```(see Example 1 in the copyright format specification).